clap_derive/src/derives/args.rs

```diff
@@ -321,6 +321,20 @@ pub fn gen_augment(
                         }
                     }
 
+                    Ty::Array => {
+                        let len = get_array_ty_len(&field.ty).unwrap();
+                        quote_spanned! { ty.span()=>
+                            .takes_value(true)
+                            .value_name(#value_name)
+                            .required(true)
+                            .number_of_values(#len)
+                            .multiple_occurrences(true)
```

Comment:
Should `len` be for number of occurrences or number of values per occurrence?  What about `VecArray`s?

Reply:
In my opinion, `len` should be total number of values, `Vec<Vec<>>` should be grouped values.

Reply:
I forget that `multiple_occurrences` enables `N * number_of_values`, will remove it currently.